.travis.yml

```diff
@@ -47,6 +50,11 @@ matrix:
       env:
         - TYPE=RELEASE
         - BUILD_TYPE=Release
+    - name: "Dev build, gcc, static library"
+      env:
+        - TYPE=DEV
+        - BUILD_TYPE=Dev
```

Comment:
We don't (yet) have a `DEV` build type for this project (that's for removing `Werror`). So let's remove these two lines until we do (just to not get confused).